src/class/builtin_classes/string.rs

```diff
@@ -1,12 +1,18 @@
 use std::{any::Any, rc::Rc};
 
-use crate::class::{Class, ClassInstance, Field, Method};
+use crate::class::{
+    class_identifier, Class, ClassIdentifier, ClassInstance, Field, Method,
+};
 
-pub struct StringClass {}
+pub struct StringClass {
+    class_identifier: ClassIdentifier,
+}
 
 impl StringClass {
     pub fn new() -> StringClass {
-        StringClass {}
+        StringClass {
+            class_identifier: class_identifier!(java / lang, String),
```

Comment:
Why write spaces around the "/"? Is there any benifit to this rather than just "java/lang"

Reply:
Because of the formatter:
```diff
cargo fmt --check
Diff in /home/florian/Documents/fh/WS23_24/Compilerbau/vm/main/src/class/builtin_classes/system.rs at line 31:
             }),
         ];
         Self {
-            class_identifier: class_identifier!(java/lang, System),
+            class_identifier: class_identifier!(java / lang, System),
             fields,
         }
     }
error: Recipe `format` failed on line 14 with exit code 1
```

Reply:
i.e., it was not a conscious decision on my side